DebugProtocol/DebugConnection.cs

```diff
@@ -339,8 +339,7 @@ void ThreadListEvent(object sender, ThreadListEventArgs args)
                 mCurrentProcess = mNewCurrentProcess;
                 mCurrentThread = mNewCurrentThread;
                 mProcesses = mAccumulateProcesses;
-                if (DebugProcessThreadChangeEvent != null)
-                    DebugProcessThreadChangeEvent(this, new DebugProcessThreadChangeEventArgs(mProcesses));
+                DebugProcessThreadChangeEvent?.Invoke(this, new DebugProcessThreadChangeEventArgs(mProcesses));

```

Comment:
If you are doing crap like this, please keep it separate from logic changes, it makes reviewing really annoying.